sale_order_invoicing_finished_task/models/sale_order.py

```diff
@@ -33,23 +33,20 @@ class SaleOrderLine(models.Model):
                  'order_id.state', 'task_ids.invoiceable')
     def _get_to_invoice_qty(self):
         lines = self.filtered(
-            lambda x: (x.product_id.type == 'service' and
-                       x.product_id.invoicing_finished_task and
-                       x.product_id.service_tracking in [
-                           'task_global_project', 'task_new_project'] and
-                       not all(x.task_ids.mapped('invoiceable'))
+            lambda x: (x.product_id.type == 'service'
+                       and x.product_id.invoicing_finished_task
+                       and x.product_id.service_tracking in [
+                           'task_global_project', 'task_new_project']
+                       and not all(x.task_ids.mapped('invoiceable'))
                        )
         )
         if lines:
             lines.update({'qty_to_invoice': 0.0})
         super(SaleOrderLine, self - lines)._get_to_invoice_qty()
 
     @api.multi
-    def _analytic_compute_delivered_quantity_domain(self):
-        # Overwrite until resolve https://github.com/odoo/odoo/issues/24038
```

Comment:
Why removing this comment if this still overwriting full domain? Can't you do this inheritable?